packages/reporter/src/errors/test-error.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import Markdown from 'markdown-it'
+import FlashOnClick from '../lib/flash-on-click'
+
+const md = new Markdown('zero', {
```

Comment:
I think this is *such* a weird API they've decided on. Passing 'zero' for the base config.